docs/source/dev/codereview.rst

```diff
@@ -0,0 +1,49 @@
+Code Review
+===========
+
+This document gives a basic overview of code reviews for the pywincffi
+projects.  All code reviews are performed on GitHub by using pull
+requests.  Information about pull requests and how to submit one can be found
+here:
+
+    https://help.github.com/articles/using-pull-requests/
+
+What branch should I use?
+-------------------------
+
+You should always base your code from the master branch unless you've been
+told otherwise.  The master branch should be considered production ready and
+other branches are usually for testing and development.
+
+What Will Be Reviewed
+---------------------
+
+    * If a new function is being added, review the `function <functions.html>`_
+      documentation and make sure the new code matches these expectations.
+    * For style issues, the default rule of thumb is to follow PEP8 unless it's
+      something Windows specific then the `function <functions.html>`_
```

Comment:
Readability: The sentence could end in "unless it's something Windows specific" and be comma/semi-colon separated from the remaining "then ...".